dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -330,6 +333,53 @@ def kurtosis(self):
                     self.match_count,
                     self._biased_kurtosis)
 
+    @staticmethod
+    def _perform_t_test(mean1, var1, n1,
+                        mean2, var2, n2):
+        results = {}
+        s_delta = var1/n1 + var2/n2
+        t = (mean1 - mean2) / np.sqrt(s_delta)
+        conservative_df = min(n1, n2) - 1
+        welch_df = s_delta ** 2 / ((var1 / n1) ** 2 / (n1 - 1) + (var2 / n2) ** 2 / (n2 - 1))
+        results.update({
+            "t-statistic": t,
+            "conservative_df": conservative_df,
+            "welch_df": welch_df
+        })
+        try:
+            import scipy.stats
+        except ImportError:
+            # Failed, so we return the stats but don't perform the test
+            warnings.warn("Could not import necessary statistical packages. "
+                          "T-test will be incomplete.", RuntimeWarning)
+            return results
+        # If scipy import was successful, now perform the *two-sided* t-test
+        conservative_t = scipy.stats.t(conservative_df)
+        conservative_p_val = (1 - conservative_t.cdf(abs(t))) * 2
+        welch_t = scipy.stats.t(welch_df)
+        welch_p_val = (1 - welch_t.cdf(abs(t))) * 2
+        results.update({
+            'results': {
+                "conservative": {
+                    'p-value': conservative_p_val
+                },
+                "welch": {
+                    'p-value': welch_p_val
+                }
+            }
+        })
+
+        alphas = [0.01, 0.05, 0.1]
```

Comment:
probably include `1e-3`

However, do we think we should do this? or just report the p-value and let them do what they want with the value?
Thoughts?

Reply:
ahh i see, you keep the p-value and the alpha comparison. Do you think it's clear to have p-value and the alpha keys at the same level vs:
```python
{
    "df": ...,
    "p-value": ...,
    "alpha":{
        0.1: ...,
        ...
    }
}
```

Reply:
Since the significance level largely depends on the context of the data and other factors, I think it slims down the differences if only the p-value was included and allowing the user to make their own interpretation. Most statistical software also only give you the p-value back.

Reply:
then let's do just p-value as you suggest